pandas/core/groupby/groupby.py

```diff
@@ -211,28 +223,40 @@ class providing the base-class of operations.
 
         The resulting dtype will reflect the return value of the passed ``func``.
 
-    >>> g[['B', 'C']].apply(lambda x: x.astype(float).max() - x.min())
+    >>> g1[['B', 'C']].apply(lambda x: x.astype(float).max() - x.min())
+         B    C
+    A
+    a  1.0  2.0
+    b  0.0  0.0
+
+    >>> g2[['B', 'C']].apply(lambda x: x.astype(float).max() - x.min())
          B    C
     A
     a  1.0  2.0
     b  0.0  0.0
 
+    The ``group_keys`` argument has no effect here because the result is not
+    like-indexed when compared to the input.
```

Comment:
I think I might be worth adding the parenthesized description of like-indexed like you did below for in the `group_keys` descriptions e.g. `because the result is not like-indexed (transformed) compared to the input`

Reply:
Thanks - I changed it to `(i.e. transformed)` with a link to the user guide on transforms in each place.